src/CLICommand/cli_command.ts

```diff
@@ -1,20 +1,18 @@
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
-import { Command } from 'commander';
-import { CliApiObject, ParsedArguments } from './cli';
-import { ERROR_EXIT_CODE } from './constants';
+import { program } from 'commander';
+import { CLIAction } from './action';
+import { CliApiObject, ParsedParameters } from './cli';
 import { Parameter, ParameterName, ParameterOverridenConfig } from './parameter';
 
 export type CommandName = string;
 export interface CommandDescriptor {
 	name: CommandName;
 	parameters: (ParameterName | ParameterOverridenConfig)[];
-	action(options: ParsedArguments): Promise<number | void>;
+	action: CLIAction;
 }
 
-const program: CliApiObject = new Command() as CliApiObject;
-program.name('ardrive');
-program.addHelpCommand(true);
-program.usage('[command] [command-specific options]');
+const programAsUnknown: unknown = program;
+const programApi: CliApiObject = programAsUnknown as CliApiObject;
```

Comment:
Double check that this is necessary

Reply:
Double checked. It's about the action callback; commander and CliApiObject has different signatures: their callback is a function, ours is a promise

Reply:
And we use that object as the default `program` object when it's not provided (as in prod, outside tests)